vagrant/aws/aws-init.sh

```diff
@@ -18,16 +18,31 @@
 # or bring up your mini Kafka cluster.
 
 # Install dependencies
-sudo apt-get install -y maven openjdk-6-jdk build-essential \
-            ruby-dev zlib1g-dev realpath python-setuptools
+sudo apt-get install -y \
+  vagrant \
+  maven \
+  openjdk-8-jdk \
```

Comment:
Are we proactively jumping to jdk8 for KIP-118 despite it not being changed elsewhere yet? This clearly must not affect anything automated since it was still using jdk6, though I'd check that if we have any AMI references anywhere they don't refer to an OS that doesn't have jdk8 by default (the Ubuntu 14.04 AMIs/images we've used some places don't, although at some point we can just bump those to 16.04 AMIs, or some other distro if people prefer).